src/main/resources/org/jenkinsci/plugins/github_branch_source/ForkPullRequestDiscoveryTrait/TrustContributors/config.properties

```diff
@@ -0,0 +1 @@
+blurb=<div class="warning">This option may be insecure in some environments. See help button.</div>
```

Comment:
There must be a better phrasing than "See help button" for the concept, but I cannot think of one.